tests/lib/snaps.sh

```diff
@@ -14,3 +14,15 @@ install_local() {
 install_local_devmode() {
     install_local "$1" --devmode
 }
+
+mksnap() {
```

Comment:
We already have functions for doing this, we could use those or special-case this for core (mk_core_snap) or something. Not urgent in this branch. Perhaps @fgimenez has an opinion.